warcio/capture_http.py

```diff
@@ -139,7 +140,8 @@ def _extract_url(self, data):
 
 # ============================================================================
 class RequestRecorder(object):
-    def __init__(self, writer, filter_func=None):
+    def __init__(self, writer, filter_func=None, warc_version=None):
+        self.builder = RecordBuilder(warc_version=warc_version)
```

Comment:
Since a writer is already passed in, I think it'd be best to just leave this using the `writer` as the builder, rather than creating another builder object in this class.